tests/backends/storage/test_fs.py

```diff
@@ -69,10 +69,16 @@ def test_backends_storage_fs_url(fs):
 
 # pylint: disable=invalid-name
 # pylint: disable=unused-argument
-def test_backends_storage_fs_list(fs):
+def test_backends_storage_fs_list(fs, monkeypatch):
     """Tests archives listing in FSStorage."""
     fs.create_dir(settings.APP_DIR)
 
+    # Force Path instantiation with fake FS
+    monkeypatch.setattr(
```

Comment:
Have you tried factoring these in a fixture? It would go nicely along the fixture moto_fs, another incompatibility between moto and pyfakefs :grimacing:

Reply:
Good Idea) Thanks) Updated this part for the swift fixture too)

Reply:
Whoops sorry I wasn't quite clear about the fixture: I should have said "alongside" haha.
I would prefer a fixture created just for this incompatibility, that would be common for all the backends. A fixture named for example "settings_fs" which would only contain 
```
    # Force Path instantiation with fake FS
    monkeypatch.setattr(
        "ralph.backends.mixins.settings",
        Settings(HISTORY_FILE=Path(settings.APP_DIR / "history.json")),
    )
```
This fixture would be called wherever necessary.
WDYT?

Reply:
Sorry for the misunderstanding; this approach looks even better) Thanks) Updated this part)